kitsune/wiki/tests/test_templates.py

```diff
@@ -192,13 +192,27 @@ def test_document_fallback_with_translation_english_slug(self):
 
     def test_document_fallback_no_translation(self):
         """The document template falls back to English if no translation
-        exists."""
+        exists. The fallback message is not shown."""
         r = revision(save=True, content='Some text.', is_approved=True)
         url = reverse('wiki.document', args=[r.document.slug], locale='fr')
         response = self.client.get(url)
         doc = pq(response.content)
         eq_(r.document.title, doc('article h1.title').text())
 
+        # Fallback message is not shown.
```

Comment:
The comment should be `Fallback message is not shown because the revision is not ready for localization`